src/App.tsx

```diff
@@ -6,6 +6,7 @@ import {
   RoadmapPage,
   MainPage,
 } from './pages'
+import LoginPage from './pages/LoginPage'
```

Comment:
you already have import from `/pages`, so please let's delete this line and use that one